### DISCUSSION TITLE
Where to publish bundles

### DISCUSSION THREAD

**Discussion by richbai90** [getporter/porter#2128] [General]
First, I'm sorry for spamming the message board. I'm working on 3 tasks at once all revolving around this tool. 😅 I do appreciate the help and activity I've seen here. 

I'm trying to test the air gap procedure as outlined in the documentation. For a simple test I wanted to publish my bundle to `hub.docker.com` but when I try to publish it I continually get 404 errors. I've tried the following as entries for the `registry` configuration.

- hub.docker.com
- hub.docker.com/richbai90
- hub.docker.com/repositories/richbai90

I'm sure I'm missing something obvious, I just don't know what it is.

**Comment by VinozzZ** [comment]
Hey there, have you checked out the documentation for publishing a bundle? https://porter.sh/distribute-bundles/#bundle-publish
From my understanding, you don't need to specify `hub.docker.com` in the `registry` field since that's the default for docker.
Maybe try `richbai90`? let me know how that goes!

**Comment by carolynvs** [comment]
The registry for Docker Hub is `docker.io` actually which is what is causing the trouble. By default Docker assumes an image reference without a registry specified is for docker.io, which is why leaving it off works too (but just for docker hub).

In your porter.yaml file you can set it either way and then call `porter publish` and it should work:

```yaml
registry: docker.io/richbai90
```

or just 

```yaml
registry: richbai90
```
